gradle.properties

```diff
@@ -1,7 +1,7 @@
 projectVersion=4.3.11-SNAPSHOT
 projectGroupId=io.micronaut
-projectDesc=Natively Cloud Native
-title=Micronaut Framework
+projectDesc=Micronaut core components: IoC, AOP, Configuration
```

Comment:
Perhaps it would be better to leave out the ": IoC, AOP, Configuration" part of the description? Somebody completely new seeing that at the top when they're looking for the HTTP server and client docs, for example, might get the impression they are in the wrong place.

I think either cut it, or expand it to something like "IoC, AOP, Configuration and Core APIs"

Reply:
How about something like "Core components supporting the Micronaut Framework." That way we don't have to remember to change it if something we list otherwise gets moved out (or in).

Reply:
I also think it might be good to briefly explain the distinction between the Framework and Core right away in the Introduction.

Reply:
+1 to @wetted's suggestion